app/helpers/location_helper.rb

```diff
@@ -2,11 +2,14 @@ module LocationHelper
   # Adapter function to munge responses from Location IQ API to our format
   def self.adapt_location_iq_response(body)
     address = body["address"]
+    # Note(jsheu): 'type' field may contain a helpful exact name match, but not
+    # necessarily. Ex: city, state, administrative, river, university, station..
+    category = body["type"]
```

Comment:
Add an rspec test for this to `adapt_location_iq_response`?

Reply:
added 👍